helm/templates/deployment-vllm-multi.yaml

```diff
@@ -61,11 +61,19 @@ spec:
       securityContext:
         {{- toYaml .Values.servingEngineSpec.securityContext | nindent 8 }}
       {{- end }}
+      hostIPC: true
+      hostPID: true
+      hostname: "{{ $.Release.Name }}-{{ $modelSpec.name }}-{{ $.Chart.Name }}"
+      subdomain: "{{ $.Release.Name }}-{{ $modelSpec.name }}-engine-service"
       containers:
         - name: "vllm"
           image: "{{ required "Required value 'modelSpec.repository' must be defined !" $modelSpec.repository }}:{{ required "Required value 'modelSpec.tag' must be defined !" $modelSpec.tag }}"
+          securityContext:
+            capabilities:
+              add:
+                - SYS_PTRACE
```

Comment:
Hey @YuhanLiu11 - the vllm deployment already has a securityContext defined on lines 163-166 of this file. I'm seeing errors with this chart in my deployment now because of `mapping key "securityContext" already defined`